src/dispatch/case/flows.py

```diff
@@ -209,48 +212,8 @@ def case_new_create_flow(
         case=case, db_session=db_session
     )
```

Comment:
Should we remove this code block? We already have it inside the case resources creation flow.
```
    individual_participants, team_participants = get_case_participants(
        case=case, db_session=db_session
    )
```

Reply:
I kept it here because of the dependency the conversation creation has on `individual_participants`. This could be handled better.